test/SlidingSyncManager-test.ts

```diff
@@ -231,6 +231,20 @@ describe("SlidingSyncManager", () => {
             );
         });
     });
+    describe("checkSupport", () => {
+        it("shorts out if the server has 'native' sliding sync support", async () => {
+            jest.spyOn(manager, "nativeSlidingSyncSupport").mockResolvedValue(true);
+            jest.spyOn(manager, "getProxyFromWellKnown")
+            await manager.checkSupport(client);
+            expect(manager.getProxyFromWellKnown).not.toHaveBeenCalled(); // We return earlier
+        });
```

Comment:
Here, I'd like to also check whether the static property has been set, any advice would be appreciated...